src/core/components/init.js

```diff
@@ -84,16 +84,18 @@ module.exports = function init (self) {
           return cb(null, true)
         }
 
-        self.log('adding assets')
         const tasks = [
           // add empty unixfs dir object (go-ipfs assumes this exists)
           (cb) => self.object.new('unixfs-dir', cb)
         ]
 
         if (typeof addDefaultAssets === 'function') {
+          // addDefaultAssets is undefined on browsers.
+          // See package.json browser config
```

Comment:
Yeah, we don't add the default assets (initial docs) in the browser to reduce the js payload.